config/envoyconfig/tls.go

```diff
@@ -0,0 +1,52 @@
+package envoyconfig
+
+import (
+	"net/url"
+	"regexp"
+	"strings"
+
+	envoy_type_matcher_v3 "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
+)
+
+func (b *Builder) buildSubjectAlternativeNameMatcher(
```

Comment:
Our docs say:

```
Example: *.corp.example.com if wild card or authenticate.corp.example.com/authorize.corp.example.com
```

I'm not sure if that actually worked before, but this code should fix it by converting it into a regular expression in Envoy.

Reply:
nice!